pkg/testutils/unset_env_var_and_restore.go

```diff
@@ -0,0 +1,28 @@
+package testutils
```

Comment:
this `testutils` package may not need to be a child of `pkg` (AFAIU, `pkg` is the "root" for the public packages)

Reply:
Agree, I guess this was just missed when the packages got moved around.